sdk/ml/azure-ai-ml/CHANGELOG.md

```diff
@@ -37,6 +37,7 @@
  - Added a `show_progress` parameter to MLClient for enable/disable progress bars of long running operations.
  - Support `month_days` in `RecurrencePattern` when using `RecurrenceSchedule`.
  - Support `ml_client.schedules.list` with `list_view_type`, default to `ENABLED_ONLY`.
+ - Registry list operation now accepts scope value to allow subscription-only based requests.
```

Comment:
this looks like it's in the wrong CHANGELOG section

Reply:
Moved to features added.